src/coreclr/src/pal/src/misc/jitsupport.cpp

```diff
@@ -34,95 +34,95 @@ PAL_GetJitCpuCapabilityFlags(CORJIT_FLAGS *flags)
 // available, using the latest kernel for release should be sufficient.
 #ifdef HWCAP_AES
     if (hwCap & HWCAP_AES)
-        CPUCompileFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_HAS_ARM64_AES);
+        CPUCompileFlags.Set(InstructionSet_Aes);
 #endif
 #ifdef HWCAP_ATOMICS
     if (hwCap & HWCAP_ATOMICS)
-        CPUCompileFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_HAS_ARM64_ATOMICS);
+        CPUCompileFlags.Set(InstructionSet_Atomics);
 #endif
 #ifdef HWCAP_CRC32
     if (hwCap & HWCAP_CRC32)
-        CPUCompileFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_HAS_ARM64_CRC32);
+        CPUCompileFlags.Set(InstructionSet_Crc32);
 #endif
 #ifdef HWCAP_DCPOP
-    if (hwCap & HWCAP_DCPOP)
-        CPUCompileFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_HAS_ARM64_DCPOP);
+//    if (hwCap & HWCAP_DCPOP)
+//        CPUCompileFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_HAS_ARM64_DCPOP);
```

Comment:
Why are these (and those below) commented out?

Reply:
These aren't currently used (they represent ISAs we don't actually support today). I would guess that is why they are commented out.

Reply:
We don't have final names for these, and they aren't used. OTOH, when it becomes time to bring them back, uncommenting should be convenient.